installer/scripts/service_control

```diff
@@ -271,7 +271,7 @@ enable_omsagent_service()
             local omsagent_initd=/etc/init.d/$OMSAGENT_WS
             cp /etc/opt/microsoft/omsagent/sysconf/omsagent.ulinux $omsagent_initd
 
-            sed -i s,WORKSPACE_ID=.*,WORKSPACE_ID=$WORKSPACE_ID,1 $omsagent_initd
+            sed -i s,%WORKSPACE_ID%,$WORKSPACE_ID,g $omsagent_initd
```

Comment:
I don't think it's a bug, but I'm curious about the different uses of sed throughout the multihoming branch; most use 1, but this uses g.